docs/source/tracking/backend-stores.rst

```diff
@@ -0,0 +1,105 @@
+  `[Azure] <http://docs.microsoft.com/azure/databricks/applications/mlflow/access-hosted-tracking-server>`_, or :ref:`the quickstart <quickstart_tracking_server>` to
+  easily get started with hosted MLflow on Databricks Community Edition.
+
+.. note::
+    In order to use :ref:`Model Registry <registry>` functionality, you must run your server using a database-backed store.
+
+.. important::
+
+    ``mlflow server`` will fail against a database-backed store with an out-of-date database schema.
+    To prevent this, upgrade your database schema to the latest supported version using
+    ``mlflow db upgrade [db_uri]``. Schema migrations can result in database downtime, may
+    take longer on larger databases, and are not guaranteed to be transactional. You should always
+    take a backup of your database prior to running ``mlflow db upgrade`` - consult your database's
+    documentation for instructions on taking a backup.
+
+.. note::
+    In Sep 2023, we increased the max length for params recorded in a Run from 500 to 8k (but we limit param value max length to 6000 internally).
+    `mlflow/2d6e25af4d3e_increase_max_param_val_length <https://github.com/mlflow/mlflow/blob/master/mlflow/store/db_migrations/versions/2d6e25af4d3e_increase_max_param_val_length.py>`_
+    is a non-invertible migration script that increases the cap in existing database to 8k . Please be careful if you want to upgrade and backup your database before upgrading.
+
+
+Deletion Behavior
+=================
+In order to allow MLflow Runs to be restored, Run metadata and artifacts are not automatically removed
+from the backend store or artifact store when a Run is deleted. The :ref:`mlflow gc <cli>` CLI is provided
+for permanently removing Run metadata and artifacts for deleted runs.
+
+
+SQLAlchemy Options
+==================
+You can inject some `SQLAlchemy connection pooling options <https://docs.sqlalchemy.org/en/latest/core/pooling.html>`_ using environment variables.
+
++-----------------------------------------+-----------------------------+
+| MLflow Environment Variable             | SQLAlchemy QueuePool Option |
++-----------------------------------------+-----------------------------+
+| ``MLFLOW_SQLALCHEMYSTORE_POOL_SIZE``    | ``pool_size``               |
++-----------------------------------------+-----------------------------+
+| ``MLFLOW_SQLALCHEMYSTORE_POOL_RECYCLE`` | ``pool_recycle``            |
++-----------------------------------------+-----------------------------+
+| ``MLFLOW_SQLALCHEMYSTORE_MAX_OVERFLOW`` | ``max_overflow``            |
++-----------------------------------------+-----------------------------+
+
+
+File Store Performance
+======================
+
+MLflow will automatically try to use `LibYAML <https://pyyaml.org/wiki/LibYAML>`_ bindings if they are already installed.
+However if you notice any performance issues when using *file store* backend, it could mean LibYAML is not installed on your system.
```

Comment:
```suggestion
However, if you notice any performance issues when using *file store* backend, it could mean LibYAML is not installed on your system.
```